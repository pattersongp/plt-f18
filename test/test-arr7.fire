func int main = () => {
        
        array[str, str] a;
        a["zeus"] = "THE GREATEST";
        a["athena"] = "THE WISEST";
        a["herakles"] = "THE STRONGEST";
        a["hermes"] = "THE CLEVEREST";

        sprint(a["zeus"]);
        sprint(a["athena"]);
        sprint(a["herakles"]);
        sprint(a["hermes"]);
        
        return 0;
}