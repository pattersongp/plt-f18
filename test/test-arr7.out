THE GREATEST
THE WISEST
THE STRONGEST
THE CLEVEREST